libr/asm/p/asm_tricore.c

```diff
@@ -15,8 +15,36 @@
 static unsigned long Offset = 0;
 static RStrBuf *buf_global = NULL;
 static ut8 bytes[128];
+enum {
+  TRICORE_GENERIC = 0x00000000,
```

Comment:
use tabs to indent

Reply:
Okay got it.
This struct i copied it from tricore.h